tests/pass-dep/tokio/tokio_mvp.rs

```diff
@@ -1,5 +1,5 @@
 // Need to disable preemption to stay on the supported MVP codepath in mio.
-//@compile-flags: -Zmiri-disable-isolation -Zmiri-permissive-provenance
+//@compile-flags: -Zmiri-isolation-error=warn -Zmiri-permissive-provenance
```

Comment:
The question I raised about this flag for the other test also applies here. I usually avoid repeating the same question multiple times, you can assume it applies to all occurrences of the same pattern.